pkg/registry/populator.go

```diff
@@ -196,7 +196,6 @@ func (i *DirectoryPopulator) loadManifests(imagesToAdd []*ImageInput, imagesToRe
 				errs = append(errs, err)
 				break
 			}
-
```

Comment:
nit: Seems like some noise in a file not being touched by this PR.

Reply:
I did this on purpose, didn't like the extra space; I'm pretty sure I introduced this in the previous patch too.

Reply:
I'd put forth the same argument as [this](https://github.com/operator-framework/operator-registry/pull/523#discussion_r527022248) comment, but if you feel strongly about it anyway it's a small PR so I guess it's not worth delaying this PR because of this.